frontend/src/main/scala/bloop/Project.scala

```diff
@@ -96,19 +95,19 @@ object Project {
   def fromConfig(file: Config.File, logger: Logger): Project = {
     val project = file.project
     val scala = project.`scala`
-    val isEmpty = scala.organization.isEmpty && scala.name.isEmpty && scala.version.isEmpty && scala.jars.isEmpty
```

Comment:
It did not work for me before because it should have been `||` instead of `&&`. The new solution is much cleaner though.